.eslintrc.js

```diff
@@ -0,0 +1,260 @@
+/*
+👋 Hi! This file was autogenerated by tslint-to-eslint-config.
+https://github.com/typescript-eslint/tslint-to-eslint-config
+
+It represents the closest reasonable ESLint configuration to this
+project's original TSLint configuration.
+
+We recommend eventually switching this configuration to extend from
+the recommended rulesets in typescript-eslint. 
+https://github.com/typescript-eslint/tslint-to-eslint-config/blob/master/docs/FAQs.md
+
+Happy linting! 💖
+*/
+module.exports = {
+  env: {
+    es6: true,
+    node: true,
+  },
+  extends: ['prettier'],
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    project: './eslint.tsconfig.json',
+    sourceType: 'module',
+  },
+  plugins: [
+    'eslint-plugin-prefer-arrow',
```

Comment:
recommend we dont bring this with is https://stackoverflow.com/questions/34361379/are-arrow-functions-and-functions-equivalent-interchangeable/34361380#34361380

Reply:
Agreed